From 114bb9670b18c76dc9f10a446488c7419a70c7e5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Peter=20Kov=C3=A1=C5=99?= <PeterKovar@Eaton.com>
Date: Thu, 29 Nov 2018 12:31:42 +0100
Subject: [PATCH] Updated autogen.sh to use automake-1.16

---
 autogen.sh | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 3852efd..620a261 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -56,7 +56,10 @@ else
 fi
 
 echo -n "checking for automake >= $AUTOMAKE_REQUIRED_VERSION ... "
-if (automake-1.15 --version) < /dev/null > /dev/null 2>&1; then
+if (automake-1.16 --version) < /dev/null > /dev/null 2>&1; then
+   AUTOMAKE=automake-1.16
+   ACLOCAL=aclocal-1.16
+elif (automake-1.15 --version) < /dev/null > /dev/null 2>&1; then
    AUTOMAKE=automake-1.15
    ACLOCAL=aclocal-1.15
 elif (automake-1.14 --version) < /dev/null > /dev/null 2>&1; then
